assets/css/style.css

```diff
@@ -415,8 +413,216 @@ footer a {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  main {
+    width: 100%;
+    background-image: url("/assets/images/desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  header {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .nav-branding {
+    display: none;
+  }
+
+  .humburger {
+    display: none;
+  }
+
+  .navMenu {
+    display: flex;
+    align-items: center;
+  }
+
+  .navItems {
+    list-style: none;
+    display: flex;
+  }
+
+  .navListItem {
+    padding: 4%;
+  }
+
+  .navItems li a {
+    text-decoration: none;
+  }
+
+  .headline {
+    background-image: none;
+    display: flex;
+    justify-content: center;
+    justify-items: center;
+    align-items: center;
+  }
+
+  .headline h1 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 4.8vw;
+    line-height: 125%;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin-top: 5%;
+    margin-left: 0;
+  }
+
+  .greenTxt {
+    margin: 0;
+  }
+
+  p {
+    font-size: 2vw;
+    margin: 0;
+    width: 60%;
+  }
+
+  .socialMedia {
+    display: flex;
+    margin-block: 0;
+  }
+
+  .moreWorks a img {
+    display: flex;
+    width: 40px;
+    height: 40px;
+    left: calc(50% - 40px / 2 + 0.5px);
+    bottom: calc(-38% - 40px / 2 + 0.5px);
+    position: relative;
+  }
+
+  .moreWorks h2 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 4vw;
+    line-height: 5.2vw;
+    margin-top: 0;
+    padding-top: 0;
+    text-align: center;
+    position: relative;
+    bottom: calc(-115% - 40px / 2 + 0.5px);
+  }
+
+  .works {
+    width: 100%;
+    display: grid;
+    grid-template-columns: auto auto auto;
+    padding: 10px;
+  }
+
+  .workSkills {
+    width: 100%;
+  }
+
+  .SkillsBTNs {
+    width: 100%;
+    height: auto;
+  }
+
+  .SkillsBTNs li button {
+    display: flex;
+    flex-direction: row;
+    background: #fff;
+    border-radius: 8px;
+    color: #36b37f;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    font-weight: 600;
+    margin-top: 13px;
+    border: 1px;
+    text-align: center;
+  }
+
+  .aboutMe {
+    background-image: url("/assets/images/about-me-bg-left-desktop.svg"), url("/assets/images/about\ me-right-\ desktop.svg");
+    background-repeat: no-repeat, no-repeat;
+    background-size: auto, auto;
+  }
+
+  .aboutMe p {
+    margin-bottom: 2%;
+  }
+
+  .form-section {
+    display: flex;
+    background-image: url("/assets/images/contact\ form-left-desktop.svg"), url("/assets/images/contact-form-bg-middle-desktop.svg"), url("/assets/images/contact-form-right-desktop.svg");
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position-x: right, center, left;
+    background-position-y: bottom, bottom, bottom;
+    background-size: 7%, 4%, 10%;
+  }
```

Comment:
- [ ] KIndly make sure you align your `form` section correctly by adjusting your spacing like margins, padding, and width. 
- [ ] Make sure every content inside has the correct width to look exactly as the provided Figma design. :100: 
- [ ] Kindly reduce the form paragraph `font-size` to meet the design :wink: 


   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/186726049-b0842b84-9ff6-4e93-8198-ac62c36ab553.png)


   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186725956-d040f2eb-6b9d-437b-ad89-2233aef39e4f.png)